src/main/webapp/consentGroupReview/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ConsentGroupReview from './ConsentGroupReview';
+import '../index.css';
+
```

Comment:
Here is where we should have some kind of service call to something like `GET /api/user` which returns the current session user. Then `isAdmin` doesn't have to be passed in from the component.

Reply:
Corrected, please check comment at the bottom of the PR